ui/src/views/Tasks/routes.js

```diff
@@ -59,12 +59,12 @@ export default path => [
   },
   {
     component: TaskLog,
-    path: `${path}/:taskId/runs/:runId/logs/live/:logUrl`,
+    path: `${path}/:taskId/runs/:runId/logs/live/:name+`,
```

Comment:
What does the trailing `+` in `/:name+` do?

Reply:
It allows it to contain `/` characters, so the URLs can be nice-looking and avoid excessive urlencoding.  In double-checking where this is encoded, I see that it correctly uses encodeURI (which does not escape `/`), but that it also leaves in the long comment about react-router, which is no longer relevant.  I'll remove that comment and push.

Reply:
Oh, ignore that, it's addressed in another commit.